docs/content/docs/help/faq.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "FAQ"
+description = "Answers to frequently asked questions."
+date = 2021-05-01T19:30:00+00:00
+updated = 2021-05-01T19:30:00+00:00
+draft = false
+weight = 30
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+lead = "Answers to frequently asked questions."
+toc = true
+top = false
++++
+
+## What is a software factory?
+
+A software factory is a structured collection of related software assets that
+aids in producing computer software applications or software components
+according to specific, externally defined end-user requirements through an
+assembly process. A software factory applies manufacturing techniques and
+principles to software development to mimic the benefits of traditional
+manufacturing. Software factories are generally involved with outsourced
+software creation. [-- Wikipedia](https://en.wikipedia.org/wiki/Software_factory)
```

Comment:
At some point we should probably also add "what is a secure software factory" to make it clear how it builds ontop of the software factory definition.

Reply:
Sure thing! I could not find an official definition, so I just use the definition for a "regular" software factory.